File: 2021 maj/c++/2.cpp
#include <iostream>
#include <fstream>
#include <string>
using namespace std;


int main()
{
    ifstream in("instrukcje.txt");
    
    string s;
    char c;
    pair <string, int> ans = {"", 0};
    string prv = "";
    int cur = 0;
    while (in >> s >> c)
    {
        if (prv == s)
        {
            cur++;
            if (cur > ans.second)
                ans = {s, cur};
        }
        else cur = 1;

        prv = s;
    }
    cout << ans.first << ' ' << ans.second << '\n';
}

File: 2021 maj/python/2.py
with open("instrukcje.txt", "r") as f:
    inst = f.read().strip().split('\n')

ans, prv, cur = ["", 0], "", 0
for line in inst:
    s, c = line.split()
    
    if prv == s:
        cur += 1
        if cur > ans[1]:
            ans = [s, cur]
    else:
        cur = 1

    prv = s
print(*ans)